store/sqlstore/session_store.go

```diff
@@ -58,7 +58,15 @@ func (me SqlSessionStore) Save(session *model.Session) (*model.Session, error) {
 		return nil, err
 	}
 
-	if err := me.GetMaster().Insert(session); err != nil {
+	query, args, err := me.getQueryBuilder().
+		Insert("Sessions").
+		Columns("Id", "Token", "CreateAt", "ExpiresAt", "LastActivityAt", "UserId", "DeviceId", "Roles", "IsOAuth", "ExpiredNotify", "Props").
+		Values(session.Id, session.Token, session.CreateAt, session.ExpiresAt, session.LastActivityAt, session.UserId, session.DeviceId, session.Roles, session.IsOAuth, session.ExpiredNotify, model.MapToJSON(session.Props)).
```

Comment:
For MapToJSON, let's do a json.Marshal manually and return if that throws an error. And then pass `string([]byte)` to the query.

Look at job_store.go/Save